modules/openapi-generator/src/main/resources/cpp-pistache-server/api-source.mustache

```diff
@@ -70,14 +68,7 @@ void {{classname}}::{{operationIdSnakeCase}}_handler(const Pistache::Rest::Reque
     try {
     {{#hasBodyParam}}
     {{#bodyParam}}
-      nlohmann::json request_body = nlohmann::json::parse(request.body());
-    {{^isPrimitiveType}}{{^isContainer}}
-      {{paramName}}.fromJson(request_body);
-    {{/isContainer}}{{/isPrimitiveType}}{{#isContainer}}  {{paramName}} = {{#isListContainer}}  {{prefix}}ArrayHelper{{/isListContainer}}{{#isMapContainer}}  {{prefix}}MapHelper{{/isMapContainer}}::fromJson<{{items.baseType}}>(request_body);{{/isContainer}}
-    {{#isPrimitiveType}}
-      // The conversion is done automatically by the json library
-      {{paramName}} = request_body;
-    {{/isPrimitiveType}}
+      nlohmann::json::parse(request.body()).get_to({{paramName}});
```

Comment:
Here there are some references to the old method

Reply:
some lines above

Reply:
I'll back out the helper removal.

Reply:
The helper is mainly important for primitive types. Do you know if the nlohmann::json is handling the primitive types properly?

Reply:
It does handle primitive types and std containers properly. The api files that use the helper class don't involve json though, so it might be best to put it back instead of trying something new.